tests/bluetooth/tester/src/l2cap.c

```diff
@@ -19,7 +19,7 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #include "bttester.h"
 
 #define CONTROLLER_INDEX 0
-#define DATA_MTU 264
+#define DATA_MTU (256 + BT_L2CAP_CHAN_SEND_RESERVE)
```

Comment:
Makes sense for setting the size of the data_pool, but I think this line still needs to be fixed:
`accept` : `chan->le.rx.mtu = DATA_MTU;`

That should probably be:
`rx.mtu = DATA_MTU - BT_L2CAP_CHAN_SEND_RESERVE`

In which case the name DATA_MTU seems to be misleading, I'd update it to `DATA_BUF_SIZE`

Reply:
fixed as suggested